GooglePlayInstant/Editor/QuickDeploy/QuickDeployConfig.cs

```diff
@@ -114,6 +116,7 @@ internal EditorConfiguration LoadEditorConfiguration(string editorConfigurationP
         [Serializable]
         public class EditorConfiguration
         {
+            public string assetBundleUrl;
```

Comment:
I thought we were going to pull the URL from the serialized value in the MonoBehaviour. Since this value is needed by the runtime, are we duplicating how we're storing this value now?

I would prefer to have a single "source of truth" for each value.

Reply:
I think it makes sense to have two different values here. 

* One represents "the last url you entered into the loading screen window field". It's global,  hidden from the user and it's only use is to pre-populate that field.

* The one in the MonoBehaviour represents "What asset bundle url should I load at Start when this scene is loaded" and it's transparent to the user by virtue of being in the scene.

Theoretically we could use the asset bundle url in the scene to pre-populate that field, but even then you run into cases where you can't access it, for example where the loading scene isn't loaded in the hierarchy. At that point we would need to fallback to a version that's stored somewhere we can access ie. the editor config.

Reply:
Of course if we didn't have the URL in our window, but instead depended on the inspector this wouldn't be an issue...

This at least deserves a comment saying how this value is used, and that it is not the value that will be used at runtime. Possibly even refer to the class/field that is used at runtime?

Reply:
Resolved